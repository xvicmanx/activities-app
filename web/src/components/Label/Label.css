.Label {
  display: inline-block;
  background-color: #ccc;
  padding: 5px;
  margin: 0 5px;
}
