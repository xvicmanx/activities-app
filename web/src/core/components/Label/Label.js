import React from 'react';
import { Tag } from 'react-bulma-components';

const Label = (props) => {
  return <Tag {...props} />;
};

Label.Group = Tag.Group;

export default Label;
