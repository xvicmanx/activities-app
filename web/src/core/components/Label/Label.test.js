import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Label from './Label';

describe('Label', () => {
  it('renders properly', () => {
    const renderer = new ShallowRenderer();
    const result = renderer.render(<Label>Test</Label>);
    expect(result).toMatchSnapshot();
  });
});
