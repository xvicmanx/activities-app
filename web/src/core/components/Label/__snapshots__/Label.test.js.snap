// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label renders properly 1`] = `
<Tag
  remove={false}
  renderAs="span"
  rounded={false}
>
  Test
</Tag>
`;
